django/contrib/admin/static/admin/css/vendor/select2/select2.css

```diff
@@ -0,0 +1,482 @@
+.select2-container {
```

Comment:
Not sure if the concerns of [#26474](https://code.djangoproject.com/ticket/26474) applies to CSS.